testing/scenario_app/ios/Scenarios/ScenariosUITests/StatusBarTest.m

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "StatusBarTest.h"
+
+@implementation StatusBarTest
+
+- (void)setUp {
+  [super setUp];
+  self.continueAfterFailure = NO;
+
+  self.application = [[XCUIApplication alloc] init];
+  self.application.launchArguments = @[ @"--tap-status-bar" ];
+  [self.application launch];
+}
+
+- (void)testTapStatusBar {
+  if (@available(iOS 13, *)) {
+    XCUIApplication* systemApp =
+        [[XCUIApplication alloc] initWithBundleIdentifier:@"com.apple.springboard"];
```

Comment:
😮 so they straight up moved the status bar out of the app and added some special hook to the uiscrollview to be able to catch it?

Reply:
The hook for uiscrollviews always existed (since like ios 2.0), they didn't break that one.